src/SignalR/common/SignalR.Common/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <NoWarn>$(NoWarn);CS3021</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why not add this to the common refs props?

Reply:
You are right.